src/functions/api.js

```diff
@@ -6,3 +6,19 @@ export function getReviews() {
       return res.data.reviews;
     });
 }
+export function getCategories() {
+  return axios
+    .get(`https://kats-nc-games.herokuapp.com/api/categories`)
+    .then((res) => {
+      return res.data.categories;
+    });
+}
+export function getCategoryReviews(category) {
+  return axios
+    .get(
+      `https://kats-nc-games.herokuapp.com/api/reviews/?category=${category}`
+    )
+    .then((res) => {
+      return res.data.reviews;
+    });
+}
```

Comment:
i posted something on slack about a feature of axios that can add *optional* queries to an API call!

These functions could be combined